beta/tests/conftest.py

```diff
@@ -67,3 +73,13 @@ def metrics_dump_dir(request):
 @pytest.fixture(scope="module")
 def dataset_dir(request):
     return request.config.getoption("--data")
+
+
+@pytest.fixture(scope="session")
+def openvino(request):
+    return request.config.getoption("--run-openvino-eval")
+
+
+@pytest.fixture(scope="module")
+def ov_data_dir(request):
+    return request.config.getoption("--ov-data-dir")
```

Comment:
This and other files. Please make sure that they all are following POSIX standard: all files must terminated with newline.
https://pubs.opengroup.org/onlinepubs/9699919799/basedefs/V1_chap03.html#tag_03_206

Reply:
BTW, this should have been detected by pylint.

Reply:
Hm.. it seems we have not pylint check against TF sources... that is not good :( I'm going to care this matter in no time.

Reply:
Don't know why newline is not displayed here, but it's presents in files, pylint passed.